spec/paths/3dsecure.yaml

```
@@ -2,6 +2,7 @@ get:
   tags:
     - 3D Secure
   summary: Retrieve a list of ThreeDSecure entries
+  operationId: get3DSecureEntries
```

```suggestion
  operationId: get3DSecureEntryCollection
```
Let's use resource name in the singular form, in some cases plural is not avaialable.